bin/hackney/.gitignore

```diff
@@ -0,0 +1,10 @@
+venv
```

Comment:
Would you mind having the python bit of this in the fixmystreet.com repo? Just as it's totally separate, and that's deployed alongside anyway.

Reply:
That's a good idea. It's a one-time thing anyway and has already served its purpose (as we'll be promoting the categories from staging to live).